File: app/src/main/java/com/github/android/lvrn/lvrnproject/view/fragment/entitieslist/core/notebookslist/NotebooksListFragment.java
package com.github.android.lvrn.lvrnproject.view.fragment.entitieslist.core.notebookslist;

import com.github.valhallalabs.laverna.persistent.entity.Notebook;
import com.github.android.lvrn.lvrnproject.view.fragment.entitieslist.EntitiesListWithSearchFragment;

/**
 * @author Vadim Boitsov <vadimboitsov1@gmail.com>
 */

public interface NotebooksListFragment extends EntitiesListWithSearchFragment {

    void openNotebook(Notebook notebook);
}

File: app/src/main/java/com/github/android/lvrn/lvrnproject/view/fragment/entitieslist/core/notebookslist/NotebooksListFragment.kt
package com.github.android.lvrn.lvrnproject.view.fragment.entitieslist.core.notebookslist

import com.github.android.lvrn.lvrnproject.view.fragment.entitieslist.EntitiesListWithSearchFragment
import com.github.valhallalabs.laverna.persistent.entity.Notebook

/**
 * @author Vadim Boitsov <vadimboitsov1></vadimboitsov1>@gmail.com>
 */
interface NotebooksListFragment : EntitiesListWithSearchFragment {
    fun openNotebook(notebook: Notebook)
}
